src/databaseSvc/settings.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     server_host: str = '0.0.0.0'
-    server_port: int = DEPLOY_CONNECTION_PORT002
+    server_port: int = 8080

```

Comment:
With that we will broke dev/main environment. We need to rewrite deploy.sh

Reply:
Yes, I think deployment things in your scope)0)